include/websock.js

```diff
@@ -175,7 +175,7 @@ function Websock() {
             }
 
             if (this._websocket.bufferedAmount < this.maxBufferedAmount) {
-                if (this._sQ.length > 0) {
+                if (this._sQ.length > 0 && this._websocket.readyState === 1) {
                     this._websocket.send(this._encode_message());
```

Comment:
It would be better to compare against the ready state constants rather than a literal 1. E.g. WebSocket.OPEN